pacemaker.spec.in

```diff
@@ -404,6 +410,10 @@ rm -rf %{buildroot}
 
 %post
 /sbin/chkconfig --add pacemaker || :
+%if %{with cman} && %{cman_native}
+# see https://bugzilla.redhat.com/1116401
+/usr/sbin/ccs_update_schema >/dev/null 2>&1 || :
+%endif
```

Comment:
>  especially since install time is not really when this should happen

That's your opinion and I disagree. The value of doing that right away (cf. statically compiled vs. dynamic languages or standard vs. lazy evaluation)
- at the point of successful `pacemaker` installation, schema already reflects that and there is no extra processing (possibly failing, which might be very unfortunate!) pending, also speeding `service cman start` up a bit
- thanks to previous, there's no risk that keen checking prior to starting `cman` service amd possibly indicating false positive when `ccs_config_validate -u` is used, occurs
- similarly (and more realistically), calling something like `ccs --addfencedev pcmk-redirect agent=fence_pcmk` (note that there's not `-i` switch that `pcs` normally adds to `ccs` invocation, otherwise there would be a show stopper already
- uniform treatment: every other component contributing to resulting `cluster.rng` does it (for good reasons and listed above)

Easily removable blurb consisting of 4 lines incl. 1 line of comment is not what I call clutter.